actors/builtin/multisig/multisig_actor.go

```diff
@@ -14,39 +14,23 @@ import (
 	hamt "github.com/ipfs/go-hamt-ipld"
 )
 
-type InvocOutput = vmr.InvocOutput
-type Runtime = vmr.Runtime
-
-var AssertMsg = autil.AssertMsg
-var IMPL_FINISH = autil.IMPL_FINISH
-
 type TxnID int64
 
 type MultiSigTransaction struct {
-	Proposer   addr.Address
-	Expiration abi.ChainEpoch
+	ID TxnID
```

Comment:
Storing the ID inside the structure is very likely unnecessary. From inspection of the Lotus code, it's unused. I suggest removing it.